pre-aio-node-with-zuul-ref.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
```

Comment:
Rename this script to a more reasonable name e.g. test_vgpu.sh

Reply:
OK